### ISSUE TITLE
Feature request: sourcemaps in separate files.

### ISSUE THREAD

**Issue by ruud** [Polymer/polymer-build#285]
### Description
The Polymer build profides an option to create sourcemaps in the build. This is done using the:

file: polymer.json
```
sourcemaps: true
```
This works fine, but the sourcemaps are included in the fragments, resulting in very large files sizes, as large as +300%

#### Actual Results
Could you add an extra setting to create separate sourcemaps files, like *.map files. That would be most welcome.

**Comment by usergenic** [comment]
100% in favor of this.  I'll look into whats necessary as we wrap up build pipeline tooling for polymer 3 in the coming couple of weeks, since I need to update the sourcemaps code anyways.  Separate files are definitely preferable.

**Comment by ruud** [comment]
Does that mean its only supported in Polymer 3?

**Comment by usergenic** [comment]
No.  Bundler and tooling will continue to support Polymer 2.  I'm just saying there's a lot of churn in the codebase to prepare Bundler to fully support Polymer 3 (more specifically, to support ES6 module bundling) and so the sourcemaps code has to be updated along with that.

**Comment by ruud** [comment]
I understand. Thanks for explaining.

**Comment by ruud** [comment]
moved to https://github.com/Polymer/tools/issues/420
